modules/340-monitoring-kubernetes/images/kubelet-eviction-thresholds-exporter/exporter/main.go

```diff
@@ -286,55 +283,11 @@ func getKubeletRootDir() (string, error) {
 }
 
 func getRuntimeRootDir(runtime string) (string, error) {
-	switch runtime {
-	case "containerd":
+	if runtime == "containerd" {
 		return getContainerdRootDir()
-	case "docker":
-		return getDockerRootDir()
 	}
```

Comment:
For kubernetes version prior to 1.24 we have ability to use NotManager docker nodes, so exporter should work with those nodes. I think we shouldn't remove docker support here